server.js

```diff
@@ -218,7 +220,7 @@ Game.prototype.sendInvitation = async function(player, subject)
         console.log('sendInvitation to', player.name, 'subject', subject);
         console.log('link: ', gameLink);
 
-        if (smtp !== undefined) {
+        if (smtp === undefined) {
```

Comment:
Please merge from my repository as this is a bug that was recently fixed.

Reply:
done